web/frontend/src/pages/form/components/AddQuestionModal.tsx

```diff
@@ -124,7 +126,54 @@ const AddQuestionModal: FC<AddQuestionModalProps> = ({
         return;
     }
   };
-
+  /*type renderQuestionModalProps = {
```

Comment:
what is that ?

Reply:
It was a tentative of modularization but it didn't work well

Reply:
so it can be removed